src/Mvc/test/WebSites/BasicWebSite/RazorComponents/Throws.razor

```diff
@@ -0,0 +1,10 @@
+﻿@page "/components/throws"
+    
```

Comment:
Note that I didn't include a test for the case where a component throws during construction/DI/parameters. That results in https://github.com/aspnet/AspNetCore/issues/8190